src/Interfaces/IInfluxSeries.cs

```diff
@@ -7,11 +7,11 @@ namespace AdysTech.InfluxDB.Client.Net
     /// </summary>
     public interface IInfluxSeries
     {
-        /// <summary>
-        /// Read only List of ExpandoObjects (in the form of dynamic) representing the entries in the query result 
-        /// The objects will have columns as Properties with their current values
-        /// </summary>
-        IReadOnlyList<dynamic> Entries { get; }
```

Comment:
Instead of changing this public interface property, can you add a new property, which then returns the `IReadOnlyList<Dictionary<string, object>>`? and may be use this in the main methods, so that internally we won't use dynamic, but if someone wants a dynamic exposed we can satisfy existing service contracts